packages/apps/dev-wallet/src/pages/home/home-page.tsx

```diff
@@ -14,9 +16,12 @@ export function HomePage() {
     profile,
   } = useWallet();
   const { activeNetwork } = useNetwork();
+  const { setLayoutContext } = useContext(LayoutContext);
+
   if (!isUnlocked) {
     return <Navigate to="/select-profile" replace />;
   }
+  profile && setLayoutContext({ accentColor: profile?.accentColor });
```

Comment:
this causes re-rendering! this should be wrapped within a useEffect, also its better to handled it in the layoutProvider itself then it will be applied to all pages automatically. 
So basically you need to update LayoutProvider like this
```TypeScript
const { profile } = useWallet();
const [accentColor, setAccentColor] = useState(profile?.accentColor ?? defaultAccentColor);
  
useEffect(()=>{
   setAccentColor(profile?.accentColor ?? defaultAccentColor,)
[profile])

return <LayoutContext.Provider value={accentColor}>
      {children} 
</LayoutContext.Provider>

```

Reply:
Updated!
I still left the function to update the color from the app because we want it to see what color we prefer before saving it to the profile.